fathom/src/lang/core/semantics.rs

```diff
@@ -54,7 +57,8 @@ pub enum Head {
     Error,
 }
 
-/// An eliminator that cannot be reduced further is 'stuck' on some [`Head`].
+/// An eliminator that cannot be reduced further due to being stuck on some
+/// 'stuck' on some [`Head`].
```

Comment:
I think there's just a bit of brokenness in the wording here.

```suggestion
/// An eliminator that cannot be reduced further due to being
/// 'stuck' on some [`Head`].
```

Maybe like that?

Reply:
Oh woops hah

Reply:
I think that was fixed in the later documentation commit

Reply:
I think i might have somehow been seeing an outdated version – perhaps this can be ignored :sweat_smile:

Reply:
Yeah, I changed the wording in cbfa5844227bc059a2d176a350709359f065c06